cmd/logger/logger.go

```diff
@@ -273,32 +273,55 @@ func hashString(input string) string {
 	return hex.EncodeToString(checksum)
 }
 
+// LogType type
+type LogType int8
```

Comment:
We could perhaps just use strings as a direct type because integer types are useful when we want to print multiple log types together. That seems like not our idea here. 

```
type Kind string

const (
    Minio Kind = "MINIO"
    Application Kind = "APPLICATION"
    All Kind = "ALL"
)
```

Reply:
So when using it looks like `func logThis(k logger.Kind)`